nginx-docker-builder/README.md

```diff
@@ -86,15 +98,16 @@ $ ./scripts/nginxWithAgentStart.sh start
 $ ./scripts/nginxWithAgentStart.sh stop
 ```
 
-3. After startup NGINX instances will register to NGINX Instance Manager / NGINX SaaS console and will be displayed on the "instances" dashboard
+3. After startup NGINX instances will register to NGINX Instance Manager / NGINX One console and will be displayed on the "instances" dashboard
 
 ### Running the docker image on Docker
 
 1. Start using
 
 ```
 docker run --rm --name nginx -p [PORT_TO_EXPOSE] \
-        -e "NIM_HOST=<NGINX_INSTANCE_MANAGER_FQDN_OR_IP>" \
+        -e "NGINX_LICENSE=<NGINX_JWT_LICENSE_TOKEN>" \
+        -e "NIM_HOST=<NGINX_CONTROL_PLANE_FQDN_OR_IP>" \
```

Comment:
There's a mix of `NMS_` and `NIM_` env vars being used in here. It's a bit confusing and doesn't seem necessary. The NGINX Agent has been using `NGINX_AGENT_` env var prefix for several releases now https://docs.nginx.com/nginx-agent/configuration/configuration-overview/#cli-flags-and-environment-variables
And neither of these make it apparent that the NGINX One Console is supported.